### PR TITLE
Add tests for classes in Query module

### PR DESCRIPTION
**Pull request by henare**
# What does this do?

Adds tests for all the classes in the Query module.

# Why was this needed?

These classes were missing tests.

# Relevant Issue(s)

# Implementation notes

These are mostly copied from their equivalent Page tests. This might seem like duplication but by having tests closer to the object it should allow us to change their behaviour with more confidence and certainty.

# Screenshots

# Notes to Reviewer

# Notes to Merger

### CODE REVIEW

**Review by tmtmtmtm — COMMENTED** [review]
I'm afraid I ran out of time to properly look at this today; so unfortunately this is much more drive-by comment than proper review. My first reaction however is indeed, as you mention, to raise an eyebrow at the duplication here. 

I think this approach of moving tests closer to what they're testing is definitely good, but I think in this case we would indeed actually want to _move_ them, not just copy them. Now that the Query classes are tested, there's much less need to test the same things in the Page classes — those tests should, I suspect, become very minimal as a result of this.

(And to be totally clear, even if that is what we decide to do, that doesn't necessarily mean we need to do it in _this_ PR — there are always things we can improve, and we should never let the best be the enemy of progress. I'm generally always OK with merging things that are better than we currently have, as long as they also include `TODO:` comments to things we know we also want to improve (ideally explicitly referencing the Github Issue too). That way we can still make steady progress, but also have a good explicit awareness of things that we're promising to pay off later.)

**Comment by henare** [comment]
Updated due to #17 being merged.

**Comment by henare** [comment]
>And to be totally clear, even if that is what we decide to do, that doesn't necessarily mean we need to do it in this PR — there are always things we can improve, and we should never let the best be the enemy of progress.

This is really good to know. I'm still pretty sensitive to doing too much in PRs and overwhelming reviewers 🙂 

Removing the duplication should be pretty straightforward and is a good win. I'll add it to this PR and hope that it doesn't make this too hard to review.

**Comment by henare** [comment]
Thanks for the drive-by comment anyway @tmtmtmtm, that helped improve this 😄 It's ready for review now.

File: t/page/country.rb [hunk 1]
```diff
@@ -3,63 +3,12 @@
...
 
-def country_page(qid)
-  country = Query::CountryInfo.new(id: qid).data
-  divisions = Query::CountryDivisions.new(id: qid).data
-  cities = Query::CountryCities.new(id: qid).data
-  Page::Country.new(country: country, divisions: divisions, cities: cities)
-end
-
 describe 'Estonia Page' do
-  around { |test| VCR.use_cassette('Estonia Page', &test) }
```
Comment (tmtmtmtm) [inline t/page/country.rb:14]: looks like this cassette should be deleted now

**Review by tmtmtmtm — APPROVED** [review]
Woot! Yep, that's much cleaner now. There's one dangling VCR cassette that I'm pretty sure should also be deleted. I'll push an extra commit to remove that, and then merge.

File: t/page/country.rb (hunk 1 above)
Comment (henare) [reply t/page/country.rb:14]: Nice catch - thanks for finding, fixing, and merging 🚀

**Pull request merged**